registry/scheduledTasks.xml

```diff
@@ -59,4 +59,8 @@
 		<descr>Process the usage stats logs and load the numbers into the DB tables.</descr>
 		<frequency day="0"/>
 	</task>
+	<task class="lib.pkp.classes.task.ProcessQueueJobs">
```

Comment:
Needs changing to: `PKP\task\ProcessQueueJobs`

Reply:
Done.